static/lang/re-en.json

```diff
@@ -1697,6 +1697,11 @@
                 "Deteriorated": "Deteriorated"
             }
         },
+        "SpellEffects": {
+            "DragonClaws": {
+                "Label": "Dragon Claw"
+            }
+        },
```

Comment:
This should go under SpecificRules, instead of adding a new top level key

Reply:
Changed